images/dind/node/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN dnf -y update && dnf -y install\
 # with the newer release. (This can go away when the base image is upgraded to
 # OVS 2.8 prerelease or release versions and include OVN sub-packages)
 RUN dnf -y install dnf-plugins-core &&\
- dnf -y copr enable leifmadsen/ovs-master &&\
+ dnf -y copr enable danw/origin-dind-ovs-master &&\
```

Comment:
What OVS do you need that's not in the CentOS PaaS SIG repos we're using elsewhere?

Reply:
Something relatively recent; OVS 2.8 (for some conntrack stuff) + OVN sub-packages (for OVN obviously).  We need to wait for F26 to get moved to OVN 2.8, and then we'll get both 2.8 and the OVN sub-packages when we switch the DIND images to F26 base.

Reply:
Oh, and the DIND images are based on Fedora, not CentOS, so...

Reply:
Actually, let me move the build from danw/origin-dind-ovs-master over to danw/origin-dind-ovs and we can use that. I don't plan to keep that COPR up to date with OVS master so it's just confusing to have "master" in the name.

> We need to wait for F26 to get moved to OVS 2.8

I'm not sure there's any plan to do that... I think we're waiting for F27 at this point. (This OVS build is a rebuild of the current rawhide RPM.)

> Oh, and the DIND images are based on Fedora, not CentOS, so...

FWIW I have a branch that switches dind over to CentOS (a spinoff of the old ansible-install-into-dind work). It makes things slightly less convenient since some packages you might want for debugging aren't available on CentOS (and even with EPEL you're still getting much older versions of things). But maybe it's a net improvement because it's more like real OCP environments?

Reply:
> Actually, let me move the build from danw/origin-dind-ovs-master over to danw/origin-dind-ovs and we can use that.

(this is done)